images/scripts/configure_daos.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Copyright 2021 Google LLC
+# Copyright (C) 2021 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#set -x
+
+if [[ $# != 2 ]]; then
+    echo This command expects 2 arguments
+    echo "configure_daos.sh  <number_of_server_instances> \"daos-server-0001[-XXXX]\""
+    exit
+fi
+
+instances=$1
+servers=$2
+
+if [ `hostname` != "daos-server-0001" ]; then
```

Comment:
instance_base_name is a property of the server module. It may be different from ".daos-server".